ci/do_ci.sh

```diff
@@ -97,7 +97,7 @@ if [[ "$CI_TARGET" == "bazel.release" ]]; then
   # toolchain is kept consistent. This ifdef is checked in
   # test/common/stats/stat_test_utility.cc when computing
   # Stats::TestUtil::MemoryTest::mode().
-  BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS} --cxxopt=-DMEMORY_TEST_EXACT=1"
+  BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS} --action_env=ENVOY_MEMORY_TEST_EXACT=true"
```

Comment:
test_env instead of action_env, action_env invalidate build caches.

Reply:
good point; missed that in the bazel  doc. done.